src/Single.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @template T
+ * @param {((x: T) => boolean)} predicate
+ * @returns {Any<T>}
+ */
+function Single (predicate) {
+  const output = []
+
+  if (!predicate) {
+    throw 'Source or predicate is null'
```

Comment:
This method has an overload which does not require a predicate. We can merge as-is but would need to keep this issue open to allow someone to implement the overload here.